src/dev-app/BUILD.bazel

```diff
@@ -63,3 +67,28 @@ sass_binary(
         "//src/material/core:all_themes",
     ] + MATERIAL_EXPERIMENTAL_SCSS_LIBS,
 )
+
+ts_devserver(
+    name = "devserver",
+    additional_root_paths = [
+        "npm/node_modules",
+    ],
+    port = 4200,
+    static_files = [
+        ":theme",
+        "@npm//:node_modules/@webcomponents/custom-elements/custom-elements.min.js",
+        "@npm//:node_modules/core-js/client/core.js",
+        "@npm//:node_modules/hammerjs/hammer.min.js",
+        "@npm//:node_modules/moment/min/moment-with-locales.min.js",
+        "@npm//:node_modules/rxjs/bundles/rxjs.umd.min.js",
+        "@npm//:node_modules/systemjs/dist/system.js",
+        "@npm//:node_modules/zone.js/dist/zone.js",
+        "@npm//material-components-web",
```

Comment:
Should this be the whole package? Or specific files?

Reply:
I think you are right that we should use specific files. It helps reducing runfiles that need to be transferred/symlinked and is generally more explicit. Updated!

<sup>Listing every implicit dependency explicitly in the devserver is somehow painful.. hopefully this changes in the future.. but even for now. Explicitly is always in some ways good</sup>